tests/features/migrationUI.spec.ts

```
@@ -0,0 +1,89 @@
+    await expect(page).toHaveURL(/\/d\//i);
+  });
+});
+
+test('default dashboard loads without JS crash', async ({
+  readProvisionedDashboard,
+  gotoDashboardPage,
+  page,
+}) => {
+  const jsErrors: string[] = [];
+  page.on('console', (msg) => {
+    if (msg.type() === 'error') jsErrors.push(msg.text());
```

ESLint

[error] Expected { after 'if' condition.

(curly)